prow/integ-suite-ocp.sh

```diff
@@ -59,7 +59,12 @@ build_images() {
     # use ubuntu:noble to test vms by default
     nonDistrolessTargets="docker.app docker.app_sidecar_ubuntu_noble docker.ext-authz "
 
-    DOCKER_ARCHITECTURES="${arch}"  DOCKER_BUILD_VARIANTS="${VARIANT:-default}" DOCKER_TARGETS="${targets} ${nonDistrolessTargets}" make dockerx.pushx
+    if [[ "${TEST_SUITE}" == "helm" ]]; then
```

Comment:
I think we could just set the `VARIANT` variable to `"distroless default"` in the job definition.

Reply:
It does not work because it builds the image but sets the tag like this: `image-registry.openshift-image-registry.svc:5000/istio-system/install-cni:istio-testing-distroless default` and we get an invalid image name error. But I made a change to use the same logic as the upstream does to build those images based in variant value and not if we are building or not to execute the `helm` test suite. The PR is updated with a new commit

Reply:
Are you sure. I haven't tested but the code here suggests it splits the `DOCKER_BUILD_VARIANT` variable by space: https://github.com/istio/istio/blob/master/tools/docker-builder/types.go#L201

And sorry, in my previous comment I guess I meant the `DOCKER_BUILD_VARIANT` variable, not `VARIANT`.